src/libraries/Microsoft.Bcl.Cryptography/tests/PemEncodingFindTests.cs

```diff
@@ -0,0 +1,476 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+using Xunit;
+
+namespace System.Security.Cryptography.Encoding.Tests
+{
+    // Tests for Downlevel implementation of PemEncoding. This was originally taken from the .NET 5 implementation of
+    // PemEncodingFindTests.
+    // https://github.com/dotnet/runtime/blob/4aadfea70082ae23e6c54a449268341e9429434e/src/libraries/System.Security.Cryptography.Encoding/tests/PemEncodingFindTests.cs
+    public abstract class PemEncodingFindTests
```

Comment:
Likewise this is a copy from the .NET 5 unit tests, with some changes for visibility purposes.